pom.xml

```diff
@@ -67,7 +66,27 @@
     <dependency>
       <groupId>io.rest-assured</groupId>
       <artifactId>rest-assured</artifactId>
-      <version>3.0.2</version>
+      <version>3.3.0</version>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>com.sun.xml.bind</groupId>
+          <artifactId>jaxb-osgi</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <!-- forcing 2.3.0 in dep for rest-assured to avoid systemPath issue on JDK11,
+      see https://github.com/jenkinsci/jaxb-plugin/pull/6 for more information -->
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.3.0</version>
```

Comment:
Maybe it is better to add a dependency on the JAXB Plugin and to add the lib to exclusions for rest-assured

Reply:
I don't think it's really needed here, it's a test dependency only. I would tend to agree more on scope=compile

Reply:
JAXB is only used here by `rest-assured`, which is a test scope dependency too. I would agree if JAXB was used by a Jenkins plugin, even in test scope. That is not the case, hence why I think here this fix is OK.